app_dart/lib/src/service/scheduler.dart

```diff
@@ -154,6 +154,10 @@ class Scheduler {
       log.severe('Failed to add commit ${commit.sha!}: $error');
     }
 
+    await luciBuildService.schedulePostsubmitBuilds(commit: commit, toBeScheduled: toBeScheduled);
```

Comment:
The fake datastore makes it hard to write a regression test as if I was to write one it would be very brittle and only work in this case